samples/general/mobile_nerf_rayquery/mobile_nerf_rayquery.cpp

```diff
@@ -0,0 +1,1236 @@
+/* Copyright (c) 2023-2024, Qualcomm Innovation Center, Inc. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "mobile_nerf_rayquery.h"
+#include "filesystem/legacy.h"
+#include "glm/gtx/matrix_decompose.hpp"
+#include "gltf_loader.h"
+#include "platform/platform.h"
+#include "rendering/subpasses/forward_subpass.h"
+#include "scene_graph/components/material.h"
+#include "scene_graph/components/mesh.h"
+#include "scene_graph/components/perspective_camera.h"
+
+namespace
+{
+constexpr uint32_t MIN_THREAD_COUNT = 1;
```

Comment:
This variable doesn't seem to be used anywhere in this sample.

Reply:
Removed, thanks for pointing out.